library/src/scripts/flyouts/NewPostMenu.tsx

```diff
@@ -59,47 +64,302 @@ function ActionItem({ item, style }: { item: IAddPost; style: ITransition }) {
+                break;
+            case "ArrowUp":
+                if (state.open && typeof state.focusedItem != "undefined") {
+                    dispatch({ type: "set_focused_item", item: (state.focusedItem + 1) % items.length });
+                }
+                break;
+            case "ArrowDown":
+                if (state.open && typeof state.focusedItem != "undefined") {
+                    dispatch({ type: "set_focused_item", item: (state.focusedItem - 1 + items.length) % items.length });
+                }
+                break;
+            case "A":
+            case "B":
+            case "C":
+            case "D":
+            case "E":
+            case "F":
+            case "G":
+            case "H":
+            case "I":
+            case "J":
+            case "K":
+            case "L":
+            case "M":
+            case "N":
+            case "O":
+            case "P":
+            case "Q":
+            case "R":
+            case "S":
+            case "T":
+            case "U":
+            case "V":
+            case "W":
+            case "X":
+            case "Y":
+            case "Z":
+            case "a":
+            case "b":
+            case "c":
+            case "d":
+            case "e":
+            case "f":
+            case "g":
+            case "h":
+            case "i":
+            case "j":
+            case "k":
+            case "l":
+            case "m":
+            case "n":
+            case "o":
+            case "p":
+            case "q":
+            case "r":
+            case "s":
+            case "t":
+            case "u":
+            case "v":
+            case "w":
+            case "x":
+            case "y":
+            case "z":
+                // Add some handling
+                break;
+            case "Enter":
+                if (typeof state.focusedItem !== "undefined") {
+                    const item = items[state.focusedItem];
+                    switch (item.type) {
+                        case PostTypes.LINK:
+                            window.location.href = item.action as string;
+                            dispatch({ type: "set_open", open: false });
+                            dispatch({ type: "set_button_focus", focus: true });
+                            dispatch({ type: "set_focused_item", item: undefined });
+                            break;
+                        case PostTypes.BUTTON:
+                            (item.action as () => void)();
+                            dispatch({ type: "set_open", open: false });
+                            dispatch({ type: "set_button_focus", focus: true });
+                            dispatch({ type: "set_focused_item", item: undefined });
+                            break;
+                        default:
+                            break;
+                    }
+                }
+                break;
+            default:
+                if (state.buttonFocus) {
```

Comment:
I don't understand why you added this case? As mentioned above, use the default for the search.

Reply:
Ok!